src/modules/form/util/formUtil.ts

```diff
@@ -295,10 +295,15 @@ export const updateFormItem = (
       }
     }
 
-    throw new Error(`Link ID  ${newItem.linkId} not found`);
+    return false;
```

Comment:
I could see this potentially leading to a bug where if `initialLinkId` is wrong somehow, then the item gets duplicated rather than an error being thrown. I think we plan to disallow changing the LinkID after initial save, though, so at that point we can get rid of `initialLinkId` altogether. So I think probably a  non-issue :)

Reply:
I think we should. If for some reason we need to change Link IDs we can use the JSON editor. I can't think of a compelling reason why we would want to change them, either.